.gitignore

```diff
@@ -1,243 +1,243 @@
-## Ignore Visual Studio temporary files, build results, and
```

Comment:
Do you intent to change every file's line ending? I don't mind switch, but this need to be consistent.